src/Php80/Php80.php

```diff
@@ -88,18 +88,25 @@ public static function preg_last_error_msg(): string
         }
     }
 
-    public static function str_contains(string $haystack, string $needle): bool
+    public static function str_contains(string $haystack = null, string $needle = null): bool
     {
+        // Cast to string to avoid deprecation notices
+        $needle = (string)$needle;
+
         return '' === $needle || false !== strpos($haystack, $needle);
```

Comment:
Shouldn't we cast the `$haystack` too?

Reply:
According to the unit tests it's not required, though it is for `$needle`